test/e2e/data/shared/v1beta1-capx/metadata.yaml

```
@@ -0,0 +1,45 @@
+# Copyright 2024 D2iQ, Inc. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# maps release series of major.minor to cluster-api contract version
+# the contract version may change between minor or major versions, but *not*
+# between patch versions.
+#
+# TODO(release-blocker): update this file only when a new major or minor version is released
+
+apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
+kind: Metadata
+releaseSeries:
+  - major: 0
+    minor: 1
+    contract: v1alpha4
+  - major: 0
+    minor: 2
+    contract: v1beta1
+  - major: 0
+    minor: 3
+    contract: v1beta1
+  - major: 0
+    minor: 4
+    contract: v1beta1
+  - major: 0
+    minor: 5
+    contract: v1beta1
+  - major: 1
+    minor: 0
+    contract: v1beta1
+  - major: 1
+    minor: 1
+    contract: v1beta1
+  - major: 1
+    minor: 2
+    contract: v1beta1
+  - major: 1
+    minor: 3
+    contract: v1beta1
+  - major: 1
+    minor: 4
+    contract: v1beta1
+  - major: 0
+    minor: 0
+    contract: v1beta1
```

We only need the specific version(s) we're testing against in here.
```suggestion
  - major: 1
    minor: 4
    contract: v1beta1
```

---

Done.